### PR TITLE
kernel: enable virtio-fs for arm64.

### PR DESCRIPTION
**Pull request by jongwu**
This patch add patch file for virtio-fs-v0.3 kernel to enable memory hot
remove to let virtio-fs available on arm64. Also, kernel config file for
virtio-fs-v0.3x for arm64 is offered.

Fixes: #973
Signed-off-by: Jianyong Wu <jianyong.wu@arm.com>

@liwei @grahamwhaley  @devimc  @Pennyzct

### CODE REVIEW

File: kernel/configs/arm64_kata_kvm_virtio-fs-v0.3.x [hunk 1]
```diff
@@ -0,0 +1,2763 @@
...
+#
+# Frame buffer Devices
+#
+# CONFIG_FB is not set
+# end of Frame buffer Devices
+
+#
+# Backlight & LCD device support
+#
+CONFIG_LCD_CLASS_DEVICE=m
```
Comment (devimc) [inline kernel/configs/arm64_kata_kvm_virtio-fs-v0.3.x]: I think modules are not supported in kata, right?

Comment (jongwu) [reply kernel/configs/arm64_kata_kvm_virtio-fs-v0.3.x]: yeah, thanks @devimc , I will update this.

**Comment by jongwu** [comment]
hi @devimc  is time to trigger ci?

**Comment by jongwu** [comment]
@devimc maybe we can trigger ci again to see if jenkins-ci-ubuntu-18-04 will fail.

**Comment by jongwu** [comment]
@devimc can you help trigger obs ci , thanks.

File: kernel/patches/virtio-fs-v0.3.x/0001-arm64-mm-Enable-memory-hot-remove.patch [hunk 2]
```diff
@@ -0,0 +1,454 @@
+From af857648ba026b21e04f2524a99cd311b5bc0243 Mon Sep 17 00:00:00 2001
```
Comment (jodh-intel) [inline kernel/patches/virtio-fs-v0.3.x/0001-arm64-mm-Enable-memory-hot-remove.patch]: Is this patch really ~19 years old?

Comment (jongwu) [reply kernel/patches/virtio-fs-v0.3.x/0001-arm64-mm-Enable-memory-hot-remove.patch]: Hi @jodh-intel, I don't think so. I just download this patch from linux patchwork and apply it. I think it's better to remove this line.

Comment (jodh-intel) [reply kernel/patches/virtio-fs-v0.3.x/0001-arm64-mm-Enable-memory-hot-remove.patch]: Well, I think we need all the patches to be "pristine" / vanilla, but it looks very odd as-is...

Comment (jongwu) [reply kernel/patches/virtio-fs-v0.3.x/0001-arm64-mm-Enable-memory-hot-remove.patch]: em, do you mean keep that line?  but I can't explain where that line come from.

Comment (jodh-intel) [reply kernel/patches/virtio-fs-v0.3.x/0001-arm64-mm-Enable-memory-hot-remove.patch]: @jongwu - all I mean is, if it's part of the original patch, we should keep it (even if it is "wrong"). Can you provide the URL for the patch on patchwork?

Comment (jongwu) [reply kernel/patches/virtio-fs-v0.3.x/0001-arm64-mm-Enable-memory-hot-remove.patch]: @jodh-intel , the URL is https://patchwork.kernel.org/cover/11127621/. I download the patch by click the "series" button. when I apply this patch, that line doesn't exist but will show when format it. it's strange and I think that line does not belong to the patch.

Comment (jodh-intel) [reply kernel/patches/virtio-fs-v0.3.x/0001-arm64-mm-Enable-memory-hot-remove.patch]: Ok, thanks for the reference @jongwu.

**Comment by jongwu** [comment]
@grahamwhaley @jodh-intel so, what I need to do is just adding comment for that tag in Dockerfile?

**Comment by jongwu** [comment]
thanks @jodh-intel , can you help trigger ci? @devimc

**Comment by devimc** [comment]
`ERROR: Please bump version in kernel/kata_config_version`

File: kernel/kata_config_version [hunk 3]
```diff
@@ -1 +1 @@
-68
+69
```
Comment (devimc) [inline kernel/kata_config_version]: please rebase, this should be 70

Comment (jongwu) [reply kernel/kata_config_version]: thanks @devimc

**Pull request merged**